### DISCUSSION TITLE
matter: Distributed Compliance Ledger (DCL) %C2%B7 Espressif Developer Portal

### DISCUSSION THREAD

**Discussion by giscus** [espressif/developer-portal#295] [General]
# matter: Distributed Compliance Ledger (DCL) %C2%B7 Espressif Developer Portal

The developer resources in just one place!

https://developer.espressif.com/blog/matter-distributed-compliance-ledger-dcl/

**Comment by saibotma** [comment]
Hey there, great article. However, I think the "typical workflow" is missing in the last chapter.

**Reply by georgik** [reply]
Thanks you @saibotma for finding the issue. Fixed in PR https://github.com/espressif/developer-portal/pull/372 .
Please, check the updated content.
